pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java

```diff
@@ -164,6 +166,12 @@ void processFile(FileNameWithOffset fileNameWithBeginOffset, long firstSequenceN
             writer.abort();
         }
 
+        File pendingFile = new File(fileNameWithBeginOffset.fileName);
+        if(pendingFile.exists()){
```

Comment:
If file does not exist then log the warning message and delete from pending file list

Reply:
updated.